content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,171 @@
+When included values are selected, the rule is evaluated if one or more included values are part of the command context.
+When excluded values are selected, the rule is evaluated if the excluded values are not part of the command context.
+
+For example, to create a rule that is evaluated in all branches but `main` of the `example-repository` repository, you can define the following scope:
+
+{{< img src="ci/scope_not_main_example_repository.png" alt="Rule scope for example-repository and not main branch" style="width:90%;">}}
+
+If a rule does not contain a scope, it is evaluated for all values for that scope.
+For example, if a rule does not contain the `repository` scope, it is evaluated for all repositories. If the rule scope is
+set to `always evaluate`, the rule is evaluated on all repositories and branches.
+
+{{< img src="ci/rule_scope_always_evaluate.png" alt="Rule scope for rules always evaluated" style="width:90%;">}}
+
+### Custom scope
+
+In addition to branch and repository, you can define a custom scope to filter rules that are evaluated for a specific CI pipeline.
+
+To add a custom scope when creating a rule:
+
+1. Click **+ Add Filter** and select **Custom Scope**.
+2. Define the scope name, for example, `team`.
+3. Define the scope's included or excluded values.
+
+Unliked the `branch` and `repository` scopes, custom scopes need to be provided in the `datadog-ci gate evaluate` command using the **--scope** option.
+For example, you can create a rule that is evaluated for the `example-repository` repository, but only when the team is `backend`:
+
+{{< img src="ci/rule_scope_example_repository_team_backend.png" alt="Rule scope for example-repository and team backend" style="width:90%;">}}
+
+The rule would be evaluated when this command is invoked in a CI pipeline of the `example-repository` repository:
+- `datadog-ci gate evaluate --scope team:backend`
+
+The rule would **not** be evaluated when the following commands are invoked instead:
+- `datadog-ci gate evaluate`, as it does not define any team.
+- `datadog-ci gate evaluate --scope team:api --scope team:frontend`, as it defines teams different from `backend`.
+
+## Manage rules
+
+To edit a Quality Gate rule, click the **Edit** icon to the right of the **Creator** avatar on the [Quality Gate Rules page][2].
+
+{{< img src="ci/edit_quality_gate_rule.png" alt="Edit a Quality Gates rule" style="width:90%;">}}
+
+To delete a Quality Gate rule, click the **Delete** icon next to the **Edit** button on the [Quality Gate Rules page][2]. Alternatively, you can edit the rule and click **Delete Rule**.
+
+{{< img src="ci/delete_quality_gate_rule.png" alt="Delete a Quality Gates rule" style="width:90%;">}}
+
+## Permissions
+
+The [`quality_gate_rules_write` permission][1] is required to create and edit Quality Gate rules.
+The [`quality_gate_rules_read` permission][1] is required to view Quality Gate rules.
```

Comment:
```suggestion
By default, only users with the `quality_gate_rules_write` permission can create and edit Quality Gate rules. Users with the `quality_gate_rules_read` permission can view Quality Gate rules.

For more information, see the [RBAC Permissions documentation][1].
```

Reply:
I'm ok with changing it in this way, although I wouldn't put the "By default" at the beginning

Reply:
Changed, let me know if that works